znai-docs/znai/snippets/math.md

```diff
@@ -15,5 +15,13 @@ The result will be a following math expression.
 
 In presentation mode, rendered expressions will automatically scale to make use of the screen space.
 
+Znai also supports inline LaTeX. Use single backticks instead of three to render math expressions inline.
```

Comment:
nice! 
let's add a separate page section? We will have two `block` and `inline`? or something like this. Joe may have a better name

Reply:
![Added!](https://user-images.githubusercontent.com/6273469/114479179-f8d14f80-9bcd-11eb-83b1-e3443341bf81.png)

Reply:
Last (hopefully) suggestion. Let's remove LATEX top level and make second level to be two top level sections so we can see them in the TOC on the left.